### PR TITLE
Hebrew project BenYehuda

### PR DESCRIPTION
**Pull request by imvladikon**
Added Hebrew corpus from https://github.com/projectbenyehuda/public_domain_dump

### CODE REVIEW

File: datasets/hebrew_projectbenyehuda/hebrew_projectbenyehuda.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+_ASSET_ROOT_URL = "https://raw.githubusercontent.com/projectbenyehuda/public_domain_dump/master/"
+_STORAGE_API_ROOT_URL = "https://raw.githubusercontent.com/projectbenyehuda/public_domain_dump/master/txt/"
+
+# download one by one file from github is too slow
+
+_METADATA_URL = _ASSET_ROOT_URL + "pseudocatalogue.csv"
+
+
+class HebrewProjectBenYehuda(datasets.GeneratorBasedBuilder):
```
Comment (lhoestq) [inline datasets/hebrew_projectbenyehuda/hebrew_projectbenyehuda.py:35]:
```suggestion
class HebrewProjectbenyehuda(datasets.GeneratorBasedBuilder):
```
this must be the camel case version of the dataset script name `hebrew_projectbenyehuda`

**Review by lhoestq — COMMENTED** [review]
Looks really good so far thanks !

I left a comment about the dataset class name. You'll need to regenerate the dataset_infos.json file after this change

Also it looks like the dummy_data.zip file is quite big (2MB), could you try to reduce its size please ? 

To do so feel free to take a look inside the dummy_data.zip file and remove all the unnecessary files to only keep 1 or 2 of them. In particular the `metadata` file should only contain 1 or 2 ids (one per file left)

File: datasets/hebrew_projectbenyehuda/hebrew_projectbenyehuda.py (hunk 1 above)
Comment (imvladikon) [reply datasets/hebrew_projectbenyehuda/hebrew_projectbenyehuda.py]: Hi, I fixed it, but got some strange errors from CI, like "../.local/lib/python3.6/site-packages/xlrd/__init__.py:170: XLRDError" I didn't use excel at all, probably it relates to other datasets

File: datasets/hebrew_projectbenyehuda/README.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+multilinguality:
+- monolingual
+size_categories:
+- 10K<n<100K
+source_datasets:
+- original
+task_categories:
+- other
+task_ids:
+- parsing
```
Comment (lhoestq) [inline datasets/hebrew_projectbenyehuda/README.md:19]: is it really a dataset for parsing ?

File: datasets/hebrew_projectbenyehuda/README.md [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+All these works are in the public domain, so you are free to make any use of them, and do not need to ask for permission.
+
+If you would like to give credit, please credit "Project Ben-Yehuda volunteers", and include a link to the site. We'd also love to hear about the uses you've made of this dump, as it encourages us to keep producing the dump. E-mail us with a brief description (and links, if/as appropriate) of your re-use, at editor@benyehuda.org.
+
+There are 10078 files, 3181136 lines
+
+Data Annotation: 
+
+### Supported Tasks and Leaderboards
+
```
Comment (lhoestq) [inline datasets/hebrew_projectbenyehuda/README.md:73]: can you fill the empty sections or use `[More Information Needed]` please ?

**Review by lhoestq — COMMENTED** [review]
Thanks ! The dummy data are fine now :) 

Could you take a look at my comments about the dataset card before we merge :

File: datasets/hebrew_projectbenyehuda/README.md (hunk 2 above)
Comment (imvladikon) [reply datasets/hebrew_projectbenyehuda/README.md:19]: Good question, I think It's not. The dataset is like the project dataset by Project Gutenberg. Removed it. What is the best option for task_categories except other? Tried to search for something like this at the project but pg19 doesn't have a dataset card

Comment (lhoestq) [reply datasets/hebrew_projectbenyehuda/README.md:19]: You can replace `other` by `sequence-modeling` to end up with:
```
task_categories:
- sequence-modeling
task_ids:
- language-modeling
```

File: datasets/hebrew_projectbenyehuda/README.md (hunk 3 above)
Comment (imvladikon) [reply datasets/hebrew_projectbenyehuda/README.md:72]: Yes, sure, added it

**Review by lhoestq — APPROVED** [review]
Thanks ! :)

**Comment by lhoestq** [comment]
merging since the CI is fixed on master

**Pull request merged**